eth_tester/backends/pyevm/main.py

```diff
@@ -524,6 +524,37 @@ def get_transaction_receipt(self, transaction_hash):
             is_pending,
         )
 
+    def get_fee_history(
+        self, block_count=1, newest_block="latest", reward_percentiles: List[int] = []
+    ):
+        if isinstance(block_count, int) and not 1 <= block_count <= 1024:
+            raise ValidationError("block_count must be between 1 and 1024")
+
+        if newest_block == "pending":
```

Comment:
Hmm eth-tester does support the "pending" block identifier. I do see this message in the `estimate_gas()` method but I'm not sure what that means exactly.

I just tried `eth_feeHistory` against mainnet with "pending" and it is a valid identifier for the endpoint. We should probably get rid of this validation here and just set `newest_block` to "latest" if "pending" is passed in. I think that makes sense from an API standpoint too since that's a valid block id but the header isn't a proper header yet... so we just return the latest possible block if "pending" is used.

Gut check / thoughts on that idea?

Honestly seems like it's a bug in `estimate_gas()` and we should handle this case similarly there but that's unrelated.

Reply:
Glad to hear I'm not the only one confused by that check. I think I had an issue with the pending block in one of the tests but I'll apply the logic to just go with "latest" and that will take care of it!